packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -199,7 +210,10 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 			badge = <WooCommerceBundledBadge />;
 		} else if ( isPremium ) {
 			const toolTip =
-				shouldUpgrade && preselected_style_variation
+				shouldUpgrade &&
+				( is_virtual || ! design.is_premium ) &&
+				selectedStyleVariation &&
+				selectedStyleVariation?.slug !== 'default'
```

Comment:
How do you feel about moving this out of here, to make it more readable?
E.g.

```js
const shouldShowTooltip =
  shouldUpgrade &&
  ( is_virtual || ! design.is_premium ) &&
  selectedStyleVariation &&
  selectedStyleVariation?.slug !== 'default';

// ...

const tooltip = shouldShowTooltip ? __( 'Unlock this style...' ) : undefined;
```

Reply:
Another good call! Done in https://github.com/Automattic/wp-calypso/pull/73476/commits/65889c840422813619c058da3302712c2cec51a5